plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/CodeTemplates.java

```diff
@@ -171,12 +172,12 @@ static IFolder createChildFolder(String name, IFolder parent, SubMonitor monitor
     }
     return child;
   }
-  
+
   @VisibleForTesting
-  static IFile createChildFile(String name, String template, IContainer parent, SubMonitor monitor,
-      Map<String, String> values) throws CoreException {
+  static IFile createChildFile(String name, String template, IContainer parent,
+      Map<String, String> values, SubMonitor monitor) throws CoreException {
 
-    monitor.subTask("Creating file " + name);
+    monitor.subTask("Creating file " + name); //$NON-NLS-1$
```

Comment:
this too should be localized as these messages are user viisble. You don't have to do it now, but please don't suppress the warning